packages/@uppy/tus/src/index.js

```diff
@@ -252,6 +259,24 @@ module.exports = class Tus extends BasePlugin {
         resolve(upload)
       }
 
+      uploadOptions.onShouldRetry = (err, retryAttempt, options) => {
+        if (err?.originalResponse?.getStatus() === 429) {
+          if (!this.requests.isPaused) {
+            const { done, value } = this.#retryDelayIterator.next()
```

Comment:
What if `this.opts.retryDelays` was `null`, then `this.#retryDelayIterator` might be `null` and not be an iterator and thus have no `next()` method, causing a hard error. Should `this.opts.retryDelays` be set to `[]` if the consumer passes a useless value for it?